nh_clinical/tests/__init__.py

```diff
@@ -2,27 +2,22 @@
 # -*- coding: utf-8 -*-
 
 # Base level tests
-from . import test_location
-from . import test_operations
-from . import test_user_management
-from . import test_api_demo
 from . import test_activity_extension
+from . import test_api_demo
 from . import test_base_extensions
+from . import test_location
+from . import test_operations
 from . import test_patient_placement_wizard
 from . import test_responsibility_allocation_wizard
+from . import test_user_management
 from . import test_users
 
 # Test Patient
 from . import test_patient
```

Comment:
As discussed we'll go with test sub-packages that mirror the models so it's easier to trace tests for a single model through multiple modules that are overriding and adding parts to it.